### PR TITLE
mri-analysis.md file for addition to bibliography

### PR DESCRIPTION
**Pull request by rohanbanerjee**
This page in the bibliography will contain reference materials regarding MRI analysis.

### CODE REVIEW

File: bibliography/mri-analysis.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# MRI Analysis
+
+## Fundamentals
+
+* [Introduction to Neuroimaging Analysis by Mark Jenkinson and Michael Chappel](http://www.drcmr.dk/bloch](http://www.neuroimagingprimers.org) -- physical copy of the book coming up in Lab soon!
```
Comment (jcohenadad) [inline bibliography/mri-analysis.md:5]: markdown hyperlink is incorrect-- you can verify by either clicking on 'view' on GH, or building local site (https://github.com/neuropoly/intranet.neuro.polymtl.ca/wiki#build-website-locally)

Comment (rohanbanerjee) [reply bibliography/mri-analysis.md]: I have updated the markdown hyperlink.

File: bibliography/mri-analysis.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# MRI Analysis
+
+## Fundamentals
+
+* [Introduction to Neuroimaging Analysis by Mark Jenkinson and Michael Chappel](http://www.neuroimagingprimers.org) -- physical copy of the book coming up in Lab soon!
```
Comment (jcohenadad) [inline bibliography/mri-analysis.md:5]: http://www.neuroimagingprimers.org is not a specific URL to the book...

Comment (jcohenadad) [reply bibliography/mri-analysis.md:5]: book ordered-- arrives in 2d-- so you can remove the 'coming up soon'

**Comment by jcohenadad** [comment]
you missed https://github.com/neuropoly/intranet.neuro.polymtl.ca/pull/87#discussion_r1054814230

**Comment by rohanbanerjee** [comment]
Updated. Since the lab has already bought the book, I have added the link to the example boxes of the book mentioned in the book and their official website.

File: bibliography/mri-analysis.md [hunk 3]
```diff
@@ -2,4 +2,4 @@
 
 ## Fundamentals
 
-* [Introduction to Neuroimaging Analysis by Mark Jenkinson and Michael Chappel](http://www.neuroimagingprimers.org) -- physical copy of the book can be found in the lab library.
+* [Introduction to Neuroimaging Analysis by Mark Jenkinson and Michael Chappel](http://www.neuroimagingprimers.org/examples/introduction-primer-example-boxes/) -- physical copy of the book can be found in the lab library.
```
Comment (jcohenadad) [inline bibliography/mri-analysis.md:5]:
```suggestion
* [Introduction to Neuroimaging Analysis (Mark Jenkinson and Michael Chappel)](http://www.neuroimagingprimers.org/examples/introduction-primer-example-boxes/) -- physical copy of the book can be found in the lab library.
```

**Pull request merged**